.github/workflows/coverage-report.yml

```diff
@@ -0,0 +1,44 @@
+name: 'coverage-report'
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+jobs:
+  coverage-report:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: shivammathur/setup-php@v2
```

Comment:
Here is still a version instead of commit SHA

Reply:
This commit should fix this
https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/pull/103/commits/444e1ebd01996ae9bbeb956cab9f93b7181f41ec